scylladb/sphinx-scylladb-theme
# Improve sidebar: scroll to current page & adjust height

Closes #929
Closes #928

This PR:

- Refactored the JS in classes (maintenance).
- The sidebar now scrolls to highlight the current page.
- The sidebar height was adjusted so that the footer does not overlap it.

## How to test

1. Clone the PR.
2. Build the docs.
3. Open [Examples > Versions](http://127.0.0.1:5500/examples/versions/)
4. The sidebar should scroll and highlight the opened page.
